proxyd/server.go

```diff
@@ -42,6 +42,7 @@ const (
 	defaultWSHandshakeTimeout    = 10 * time.Second
 	defaultWSReadTimeout         = 2 * time.Minute
 	defaultWSWriteTimeout        = 10 * time.Second
+	defaultCacheTtl              = 1 * time.Hour
```

Comment:
[nit] title says 2 hours but this is 1 hour - either way works for me but calling out in case this was a mistake

Reply:
ah yeah fixed the title/description - it was a changed decision and converged to 1 hour